R/CiteSeq.R

```diff
@@ -441,15 +441,15 @@ ProcessCiteSeqData <- function(seuratObj, assayName = 'ADT',
   seuratObj[["origClusterID"]] <- Idents(seuratObj)
   
   #PCA:
-  if(!HasStepRun(seuratObj, 'PCA_ADT', forceReCalc = forceReCalc)){
+  if(!HasStepRun(seuratObj, 'PCA_ADT', forceReCalc = forceReCalc) || forceReCalc){
```

Comment:
Are you sure about these?  isnt HasStepRun(forceReCalc =forceReCalc ) covering this?